setup.py

```diff
@@ -153,7 +153,12 @@ def write_version_py(filename=_VERSION_FILENAME):
         install_requires = __requires__,
         license = "BSD",
         packages = find_packages(),
-        package_data = {'': ['images/*', '*.ini',]},
+        package_data = {'': ['images/*', '*.ini',],
```

Comment:
I think this is going to include `tox.ini` in eggs, whereas I expect we just want it in the sdist.